modules/installation-user-infra-machines-advanced-customizing-live-iso.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+[id="installation-user-infra-machines-advanced-customizing-live-iso_{context}"]
+= Customizing a live {op-system} ISO image
+You can customize a live {op-system} ISO image directly with `coreos-installer iso customize`. When the ISO image boots, the customizations will be applied automatically.
+
+Customizations made to the ISO image using the `customize` subcommand are wrapped by the necessary Ignition config files and placed in `/etc/coreos/installer.d`. The install script then passes these files to the installer.
+
+.Procedure
+
+. Download the `coreos-installer` binary from the following image mirror page: https://mirror.openshift.com/pub/openshift-v4/clients/coreos-installer/latest/.
+
+. Retrieve the {op-system} ISO image and the Ignition config file, and run the following to customize the ISO image using the desired Ignition config:
++
+[source,terminal]
+----
+$ coreos-installer iso customize rhcos-<version>-live.x86_64.iso \
+    --installer-config live.ign \
+    --dest-ignition user.ign <1>
```

Comment:
If the idea is for this procedure to yield an ISO ready for automated installs, then this also requires specifying `--dest-device`.

Reply:
We should explicitly call this out: if `--dest-device` is specified, the ISO will automatically run an install.  If it isn't, the ISO will be configured for installing, but will not install automatically unless the `coreos.inst.install_dev` kernel argument is specified.